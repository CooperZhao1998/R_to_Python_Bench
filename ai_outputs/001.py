# ai_outputs/001.py  (Pandas reference) â€” writes ai_outputs/001.csv
import pandas as pd
df = pd.read_csv("data/flights.csv", parse_dates=["dep_time"])
out = (
    df.loc[:, ["carrier","dep_delay","arr_delay","dest"]]
      .loc[lambda d: d["dep_delay"].notna()]
      .loc[lambda d: (d["dep_delay"] > 10) & (d["arr_delay"] <= 60)]
      .sort_values(["carrier","dep_delay","dest"], kind="mergesort")
)
out.to_csv("ai_outputs/001.csv", index=False)
