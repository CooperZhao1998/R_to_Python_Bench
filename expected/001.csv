carrier,dep_delay,arr_delay,dest
AA,11,1,ATL
AA,12,8,LAX
AA,15,37,JFK
AA,19,38,DFW
AA,20,4,JFK
AA,30,37,JFK
AA,51,2,JFK
AA,71,52,ATL
AA,80,12,ATL
AA,94,-7,JFK
AA,94,47,LAX
AA,109,2,JFK
AA,109,19,ORD
AA,127,25,LAX
AA,134,-7,JFK
AA,169,16,ORD
B6,11,31,ATL
B6,20,3,LAX
B6,20,5,LAX
B6,22,29,LAX
B6,46,38,ORD
B6,46,3,SFO
B6,60,1,JFK
B6,72,2,JFK
B6,94,2,DFW
B6,98,38,JFK
B6,98,2,ORD
B6,131,16,ATL
B6,158,1,SFO
B6,173,16,JFK
DL,47,10,DFW
DL,59,1,DFW
DL,62,20,DFW
DL,66,10,JFK
DL,70,16,LAX
DL,71,4,ATL
DL,72,5,JFK
DL,72,5,SFO
DL,131,6,JFK
DL,146,2,SFO
DL,150,-7,JFK
DL,156,2,SFO
DL,164,-7,LAX
DL,173,-7,ATL
UA,11,25,DFW
UA,15,52,ATL
UA,19,5,ORD
UA,59,25,DFW
UA,62,12,DFW
UA,91,3,JFK
UA,91,16,LAX
UA,94,5,ATL
UA,104,12,JFK
UA,127,52,ATL
UA,134,12,DFW
UA,146,56,DFW
UA,150,4,SFO
UA,150,20,SFO
UA,167,31,DFW
UA,169,16,LAX
WN,20,16,SFO
WN,26,-7,SFO
WN,35,-7,ATL
WN,35,7,SFO
WN,66,-7,ATL
WN,66,16,ORD
WN,66,8,SFO
WN,70,29,DFW
WN,72,47,DFW
WN,100,16,JFK
WN,108,16,ATL
WN,110,-7,SFO
WN,123,6,ATL
WN,150,20,LAX
WN,177,12,ATL
